manifests/init.pp

```diff
@@ -125,6 +129,9 @@
   Boolean $enable_docker = true,
   Boolean $enable_deb = true,
 
+  Boolean $use_pulp_2_for_file = $use_pulp_2_for_file,
```

Comment:
This needs to be an explicit value, such as:
```suggestion
  Boolean $use_pulp_2_for_file = true,
```

Reply:
:man_facepalming: Duh Eric